src/shared/components/DatePicker/DatePicker.styles.ts

```diff
@@ -225,3 +225,7 @@ export const DatePickerFallback = styled(Box)`
   min-height: 30rem;
   min-width: 30rem;
 `;
+
+export const StyledSpan = styled('span')`
```

Comment:
Why do you need the span?

Reply:
The tooltip throw the warning in console, it requires an element wrapper.